config/config_cheyenne_gnu.sh

```diff
@@ -16,30 +16,23 @@ export MAKE_VERBOSE=N
 export   MAKE_CLEAN=N
 export DOWNLOAD_ONLY=N
 export STACK_EXIT_ON_FAIL=Y
-export WGET="wget -nv"
+export WGET="wget -nv --no-check-certificate"
 
 module purge
-module unuse /glade/u/apps/ch/modulefiles/default/compilers
-module use   /glade/p/ral/jntp/GMTB/tools/compiler_mpi_modules/compilers
-export MODULEPATH_ROOT=/glade/p/ral/jntp/GMTB/tools/compiler_mpi_modules
-
-module load gnu/10.1.0
-module load mpt/2.22
-module load ncarcompilers/0.5.0
-module load ncarenv/1.3
-
-# Load these basic modules for Cheyenne
-module load cmake/3.18.2
+module load gnu/11.2.0
+module load mpt/2.25
+module load cmake/3.22.0
+module load python/3.7.9
 
 # gfortran-10 compatibility flags for incompatible software
 export STACK_esmf_FFLAGS="-fallow-argument-mismatch -fallow-invalid-boz"
 export STACK_pnetcdf_FFLAGS="-fallow-argument-mismatch -fallow-invalid-boz"
 export STACK_madis_FFLAGS="-fallow-argument-mismatch -fallow-invalid-boz"
+export STACK_fms_FFLAGS="-march=core-avx2 -fallow-argument-mismatch"
 
 # Build FMS with AVX2 flags
 export STACK_fms_CFLAGS="-march=core-avx2"
-export STACK_fms_FFLAGS="-march=core-avx2 -fallow-argument-mismatch"
-
-# Patch FMS
-export STACK_fms_PATCH="cheyenne_gnu_fms_mpp_util_mpi_inc.patch"
```

Comment:
Is this no longer needed with mpt-2.25? We just recently went through this exercise and found that it was still needed for the ufs-weather-model (with mpt-2.22 and gnu 10.1.0 at least).

Reply:
The code to be updated, to leave only the functionality of Lmod module collections, and  to avoid use of different modules from those in the current "develop" branch.